source/Calamari.UnmigratedCalamariFlavours/Calamari.UnmigratedCalamariFlavours.csproj

```diff
@@ -0,0 +1,39 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net5.0</TargetFramework>
+        <Copy>true</Copy>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Sashimi.Aws.Accounts" Version="14.2.0" GeneratePathProperty="true" />
+        <PackageReference Include="Sashimi.Azure.Accounts" Version="14.2.0" GeneratePathProperty="true" />
+        <PackageReference Include="Sashimi.Aws.Common" Version="14.2.0" GeneratePathProperty="true" />
+        <PackageReference Include="Sashimi.Azure.Common" Version="14.2.0" GeneratePathProperty="true" />
+        <PackageReference Include="Sashimi.AzureAppService" Version="2.4.2" GeneratePathProperty="true" />
+        <PackageReference Include="Sashimi.AzureCloudService" Version="12.1.2" GeneratePathProperty="true" />
+        <PackageReference Include="Sashimi.AzureWebApp" Version="13.2.1" GeneratePathProperty="true" />
+        <PackageReference Include="Sashimi.AzureResourceGroup" Version="11.1.0" GeneratePathProperty="true" />
+        <PackageReference Include="Sashimi.GoogleCloud.Accounts" Version="2.1.2" GeneratePathProperty="true" />
+        <PackageReference Include="Sashimi.GoogleCloud.Scripting" Version="2.1.2" GeneratePathProperty="true" />
+        <PackageReference Include="Sashimi.Terraform" Version="11.1.0" GeneratePathProperty="true" />
+        <PackageReference Include="Sashimi.AzureServiceFabric" Version="12.1.2" GeneratePathProperty="true" />
+        <PackageReference Include="Sashimi.AzureScripting" Version="14.2.1" GeneratePathProperty="true" />
+    </ItemGroup>
+
+    <!-- Copy over sashimi nuget packages from the packages directory. -->
+    <Target Name="CopyPackages" BeforeTargets="PrepareForBuild">
+        <Copy SourceFiles="$(PkgSashimi_Aws_Accounts)\sashimi.aws.accounts.14.2.0.nupkg" DestinationFolder="$(ProjectDir)\artifacts"></Copy>
```

Comment:
This is hardcoded(!) as the filename contains the version. I couldn't find a way to do a wildcard on these, but if anyone knows a better alternative to this, would be appreciated. (although this process is temporary)

Reply:
I think given the process is temporary, we can live with it :)